### ISSUE TITLE
How to get datasets when teaching Python?

### ISSUE THREAD

**Issue by gvwilson** [merely-useful/merely-useful.github.io#5]
Getting data when teaching R is simple: use `library(datasets)` and then access datasets like `iris` directly. Is there an equally simple mechanism when teaching Python? [PyDataset](https://github.com/iamaziz/PyDataset) hasn't been updated since 2016; are there other options for installing a library that includes lots of useful datasets that *won't* require novices to figure out paths to downloaded files?

**Comment by joelostblom** [comment]
I provided an [answer to a similar SO question](https://stackoverflow.com/a/29956221/2166823) previously. In brief, `statsmodel` gives access to many (all?) of the R sample data sets, and `quilt` is a package manager for an even wider selection of data sets. `seaborn` and `scikit-learn` does not have as many sample data sets, but are convenient if those packages are already installed and used in the lecture. See the answer I linked for details, and additional options.
